src/dotnet/Program.cs

```diff
@@ -260,14 +260,12 @@ private static void PrintVersion()
 
         private static void PrintInfo()
         {
-            HelpCommand.PrintVersionHeader();
-
             DotnetVersionFile versionFile = DotnetFiles.VersionFileObject;
             var commitSha = versionFile.CommitSha ?? "N/A";
             Reporter.Output.WriteLine();
-            Reporter.Output.WriteLine("Product Information:");
-            Reporter.Output.WriteLine($" Version:            {Product.Version}");
-            Reporter.Output.WriteLine($" Commit SHA-1 hash:  {commitSha}");
+            Reporter.Output.WriteLine(".NET Core SDK (reflecting any global.json):");
+            Reporter.Output.WriteLine($" Version:   {Product.Version}");
+            Reporter.Output.WriteLine($" Commit:    {commitSha}");
             Reporter.Output.WriteLine();
             Reporter.Output.WriteLine("Runtime Environment:");
```

Comment:
Base Path also reflects global.json and should probably go under .NET Core SDK.

Reply:
For this, I need @KathleenDollard's agreement. While is makes sense because of the reflecting global.json message, we have always kept this information here.